### PR TITLE
Chinese Localization of README

### PR DESCRIPTION
**Pull request by Mrright024**
I make it! Not translated all of the content but I think it should be enough for most of the people.

### CODE REVIEW

**Comment by Mrright024** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fdnschneid%2Fcrouton%2Fpull%2F4217) for more info**.

@googlebot I signed it!

**Comment by Mrright024** [comment]
#4217

**Comment by drinkcat** [comment]
LGTM (not that I actually tried to read it ,-P), I have a mild preference for `zh_CN.md` filename as that clearly splits up Simplified vs Traditional. But I think it's ok as-is until somebody decides to write up a Traditional one.

File: README.zh.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## 简介
+
+​	Chroot是Chromium OS Universal Chroot Environment 的简写，是一系列脚本的合集，利用Linux的Chroot，在Chromebook上同时运行Chrome OS和某个Linux发行版。
+
+## Chroot介绍
+
+​	Chroot命令用来在指定的根目录下运行指令。Chroot的这种功能可以为第二系统提供一个隔离的文件系统，就像虚拟化一样，但是第二系统实际上仍然在主系统的文件系统下面工作，在进程和网络层面，chroot并没有进行隔离。
+
+​	至于详细的内容，为什么不去问问[百度搜索](https://www.baidu.com/s?wd=chroot)？
```
Comment (drinkcat) [inline README.zh.md:13]: Maybe a Wikipedia link instead? https://zh.wikipedia.org/wiki/Chroot

File: README.zh.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### 环境
+
+- **良好的**网络环境
+- 进入**开发者模式**的Chromebook，相关操作请进入[这个页面](https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices)（英文页面），点击对应的设备型号，按照*Entering Developer Mode*章节的步骤进行
+- 强烈建议安装[Crouton插件](https://goo.gl/OVQOEt)，配合`extension`或者`xiwi`目标，可以提高第二系统与Chrome OS之间的交互体验
+
+### 用法
+
+1. 你需要从[这里](https://goo.gl/fd3zc)下载Crouton脚本。~~什么？下不下来？关我什么事~~
```
Comment (drinkcat) [inline README.zh.md:25]: `~~什么？下不下来？关我什么事~~` can you remove that bit please? (we're not 100% sure how to read it, better keep it safe)

File: README.zh.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+## 示例
+
+**简单示例（安装Ubuntu LTS，使用Xfce桌面环境）**
+
+1. 下载Crouton
+2. 打开shell（`ctrl+alt+T`,在打开的窗口中输入`shell`，然后回车）。
+3. 输入`sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton`
+4. `sudo crouton -t xfce`
+5. 等吧，可以喝杯星巴克
```
Comment (drinkcat) [inline README.zh.md:47]: `星巴克` is ok, but maybe add something like `or your other preferred coffee place` ,-)

File: README.zh.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+​	使用`sudo crouton -u -n chrootname`来升级chroot环境中的系统。
+
+**安装后想添加一些安装目标？**
+
+​	使用`-u`参数来添加安装目标。
+
+​	比如，添加`xiwi`目标：`sudo crouton -t xiwi -u -n chrootname`
+
+​	上述命令会让xiwi成为默认的[X窗口方法](https://baike.baidu.com/item/X%E7%AA%97%E5%8F%A3/1471357?fr=aladdin)（原默认方法为xorg），如果想让X窗口方法继续保持默认：
```
Comment (drinkcat) [inline README.zh.md:85]: Again, maybe a wikipedia link?
https://zh.wikipedia.org/wiki/X%E8%A6%96%E7%AA%97%E7%B3%BB%E7%B5%B1

File: README.zh.md (hunk 1 above)
Comment (Mrright024) [reply README.zh.md:13]:
```suggestion
​	至于详细的内容，为什么不去问问[维基百科](https://zh.wikipedia.org/wiki/Chroot)？（或者[百度百科](https://www.baidu.com/s?wd=chroot)，如果你上维基很困难的话）

```
A Wikipedia link is a good idea. Wanna keep the Baidu link cause one may have difficulty accessing Wikipedia in mainland China.

File: README.zh.md (hunk 2 above)
Comment (Mrright024) [reply README.zh.md:25]:
```suggestion
1. 你需要从[这里](https://goo.gl/fd3zc)下载Crouton脚本。（此处需要科学上网）

```
Just a joke, will be replaced with tips.

File: README.zh.md (hunk 3 above)
Comment (Mrright024) [reply README.zh.md:47]:
```suggestion
5. 等吧，可以喝杯星巴克，或者喝点别的也行

```

File: README.zh.md (hunk 4 above)
Comment (Mrright024) [reply README.zh.md:85]:
```suggestion
​	上述命令会让xiwi成为默认的[X窗口方法](https://zh.wikipedia.org/wiki/X%E8%A6%96%E7%AA%97%E7%B3%BB%E7%B5%B1)（或者可以看[百度百科](https://baike.baidu.com/item/X%E7%AA%97%E5%8F%A3/1471357?fr=aladdin)），如果想让X窗口方法继续保持默认（原默认方法为xorg）：

```
kepp the Baidu link for the same reason above

File: README.zh.md (hunk 1 above)
Comment (drinkcat) [reply README.zh.md:13]: noted.

File: README.zh.md (hunk 4 above)
Comment (drinkcat) [reply README.zh.md:85]: gotcha.

File: README.zh.md (hunk 2 above)
Comment (drinkcat) [reply README.zh.md:25]: We're still a tad confused about `科学上网`, but should be fine ,-) Thanks.

File: README.zh.md (hunk 3 above)
Comment (drinkcat) [reply README.zh.md:47]: 比較好。

**Comment by drinkcat** [comment]
Should be good with those small fixes, thanks.

**Pull request merged**